msg/AccelGyroTempData.msg

```diff
@@ -0,0 +1,12 @@
+# Accelerometer [m/s^2]
+
+geometry_msgs/Vector3 accel
+
+# Gyroscope [rad/s]
+
+geometry_msgs/Vector3 gyro
+
+# Internal Temperature [°C]
+float32 temp
+
+int32 count
```

Comment:
this looks a little strange, I think all fields should have a comment and there should be no extra newlines between the comment and the field